examples/TemplateGenerationIterative.ipynb

```diff
@@ -6,7 +6,9 @@
    "source": [
```

Comment:
<p>Can we add <code>itk-shape</code>  to the package dependencies:</p><p> </p><p>  https://github.com/KitwareMedical/HASI/blob/daa1d72d7759f421a93f738ce44c1afffd799307/src/hasi/pyproject.toml#L11</p><p> </p><p>And change this to:</p><p> </p><blockquote>This pipeline uses the <a href="https://github.com/slicersalt/ITKShape" target="_blank">ITKShape</a> module for shape analysis.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/HASI/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>